rampwf/tests/kits/titanic_test_external_imports/submissions/starting_kit/estimator.py

```diff
@@ -0,0 +1,26 @@
+from sklearn.compose import make_column_transformer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline
+from sklearn.linear_model import LogisticRegression
+from utils import test_imports as test
```

Comment:
I would not use the `as test` and just use `test_imports` to keep the explicit name

Reply:
also please add a blank line between this import and the previous ones